src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorManager.class.php

```diff
@@ -225,42 +225,16 @@ public function HandleExternalFunctionCall($sFunctionName, $aParameters = array(
     }
 
     /**
-     * load the table editor class that is configured in cms_tbl_conf.
-     *
-     * @deprecated the hardcoded switch by table name is deprecated and can be deleted after upgrading all customers
-     *
-     * @return TCMSTableEditor
```

Comment:
See comment below. We need the return type info here.

Reply:
ok done